src/dependabot.ts

```diff
@@ -42,8 +42,13 @@ export const getOwnerAndRepo = async (
   ownerFlag: string,
   repoFlag: string
 ): Promise<{ owner: string; repo: string }> => {
+  if (ownerFlag && repoFlag) {
```

Comment:
Let's add an error if only a single on of these flags are passed (similar to `dependsOn`)